docs/guide/install.md

```diff
@@ -37,3 +37,61 @@ $ npx slidev
 > ```bash
 > echo 'shamefully-flatten=true' >> .npmrc
 > ```
+
+## Install on Docker
+
+If you need a rapid way to run a presentation, you can use the prebuilt (or even build your own) [docker](https://hub.docker.com/r/stig124/slidev) image (maintained by [stig124](https://github.com/Stig124))
+
+##### *(this is not meant as a developper environnement, as it uses only npm-fetched binaries)*
+
+Pull the image :
+
+```bash
+$ docker pull stig124/slidev:latest
+```
+
+> [podman](https://podman.io/) users can pull the image using the complete URL
+>
+> ```bash
+> $ podman pull docker.io/stig124/slidev:latest
+> ```
```

Comment:
It looks like a more opinionated tools, which users could choose whatever they prefer, I don't think we need to document this here.

Reply:
podman is the drop-in replacement to Docker, deprecating Docker on some Enterprise Linux

Reply:
like `yarn` to `npm`?

Reply:
I don't know the JS world, but pretty much yeah

Reply:
Then I'd prefer just simply:

~~~md
```bash
$ docker pull stig124/slidev:latest
```

Or

```bash
$ podman pull docker.io/stig124/slidev:latest
```
~~~